src/main/resources/db/migration/V4__send_erc20_request_table.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE blockchain_api_service.send_erc20_request (
+    id                  UUID        PRIMARY KEY,
+    chain_id            BIGINT      NOT NULL,
+    redirect_url        VARCHAR     NOT NULL,
+    token_address       VARCHAR     NOT NULL,
+    amount              NUMERIC(78) NOT NULL,
+    from_address        VARCHAR,
+    to_address          VARCHAR     NOT NULL,
+    arbitrary_data      JSON        NOT NULL,
```

Comment:
Why limit arbitrary data to `not null`?

Reply:
It's non-nullable in the request data, and we can always use an empty JSON object to represent empty state. But I will double check with @mightymatth to be sure.

Reply:
No problem, it's also ok to store empty `json`.